test/test_change_stream.py

```diff
@@ -54,6 +54,7 @@ class TestClusterChangeStream(IntegrationTest):
 
     @classmethod
     @client_context.require_version_min(4, 0, 0, -1)
+    @client_context.require_storage_engine("wiredTiger")
```

Comment:
Change streams work of storage engines other than wiredTiger. Like inMemory for example. Can you make these `@client_context.require_not_mmap` instead?